test/cases/enum.ts

```diff
@@ -6,7 +6,8 @@ export var schema = {
       "enum": ["a", "b", "c"]
     },
     "bar": {
-      "enum": [1, 2, 3]
+      "enum": [1, 2, 3],
+      "tsEnumNames": ["One","Two","Three"]
```

Comment:
For case 2 to work, it would require my enum util classes, because under the hood TS enums *are* integers, and there's nothing we can do about that until complex enum types are implemented.  The implication of this is that anyone else using our schema will be passing strings, and we will need to know what to do with them, which right now we don't (since symbolic name information is lost after compile time).

Right now, case 1 and case 3 work.